logos/src/logos/model_string_parser.py

```
@@ -0,0 +1,86 @@
+    extra = dict()
+
+    # Check string body
+    try:
+        for param in params[1:]:
+            if "policy_" in param:
+                vals = param.replace("policy_", "", count=1)
+                # Get all "k=v"-Pairs
+                for kv in vals.split("_"):
+                    k, v = kv.split("=", maxsplit=1)
+                    if k not in POLICY_FIELDS:
+                        raise ValueError("Misplaced attribute in policy fields: %s. Allowed attributes: %s", k, POLICY_FIELDS)
```

Ruff (0.14.0)

Abstract `raise` to an inner function

(TRY301)